### ISSUE TITLE
Question on table 16 description

### ISSUE THREAD

**Issue by Seeker98** [c-yn/AdaIR#18]
Thank you for your insightful work that provides novel perspectives on general-purpose IR systems. Regarding Table 16, I would like to seek clarification on two aspects:

* Based on the performance patterns shown in the table, I presume the labels "one-task," "two-task," and "three-task" directly correspond to the number of concurrent learning objectives. Could you confirm if this interpretation aligns with the experimental design?

* The table's title as "three-task setting", but in the paper, training settings are consist between 3-in-1 and 5-in-1. Could you elaborate on whether this terminology implies extra experimental settings?

**Comment by Seeker98** [comment]
Thanks for the early reply, and I'm now clear with my questions.






------------------ Original ------------------
From: Yuning Cui ***@***.***&gt;
Date: Mon,Mar 24,2025 10:02 PM
To: c-yn/AdaIR ***@***.***&gt;
Cc: Victor Chen ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [c-yn/AdaIR] Question on table 16 description (Issue #18)



   
Hi, thank you for your interest.
 
The experimental setup in Table 16 (ablations) is identical to that of ALL-IN-ONE RESULTS: THREE DISTINCT DEGRADATIONS. The difference is the number of datasets used for training.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  c-yn left a comment (c-yn/AdaIR#18)
 
Hi, thank you for your interest.
 
The experimental setup in Table 16 (ablations) is identical to that of ALL-IN-ONE RESULTS: THREE DISTINCT DEGRADATIONS. The difference is the number of datasets used for training.
